test/logging/request-obfuscator_test.js

```diff
@@ -43,15 +43,15 @@ describe('RequestObfuscator', () => {
       request.should.eql({ body: '{"id":"1485369469422","method":"importprivkey","params":["******"]}' });
     });
 
-    it('should obfuscate the private key from `request.body` when `method` is `signmessagewithprivkey`', () => {
+    it.skip('should obfuscate the private key from `request.body` when `method` is `signmessagewithprivkey`', () => {
```

Comment:
@ahmer-yasin  Are there any tests that still function or did all of them get marked with "skip"?